pkgs/development/python-modules/pyshark/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildPythonPackage, fetchPypi, py, lxml, pytest }:
+
+buildPythonPackage rec {
+  pname = "pyshark";
+  version = "0.4.2.11";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "048nnjci2ikbr7s9czfd94gfa847sbg4sxl5pkwd9hmi1qxmq3rl";
+  };
+
+  propagatedBuildInputs = [
+    py
+    lxml
+  ];
+
+  checkInputs = [
+    pytest
```

Comment:
```suggestion
    pytestCheckHook
```

Reply:
Actually nevermind this, the release tarball doesn't contain any unit tests. I added a pythonImportsCheck instead.

Reply:
Then please fetch the source from Git instead. We really want to have tests where possible in pythonPackages.

Reply:
Sure, done